src/ispypsa/config/validators.py

```diff
@@ -16,12 +16,32 @@ class NetworkConfig(BaseModel):
     nodes: NodesConfig
 
 
-class TraceConfig(BaseModel):
+class TemporalConfig(BaseModel):
+    operational_temporal_resolution_min: int
     path_to_parsed_traces: str
     year_type: Literal["fy", "calendar"]
     start_year: int
     end_year: int
     reference_year_cycle: list[int]
+    representative_weeks: list[int] | None
```

Comment:
See comment about config organisation. Could create an `AggregationConfig` class with optional fields that are concrete types or refer to validation classes for each type of aggregation

Reply:
Made this change.